allure-python-commons/src/_allure.py

```diff
@@ -70,6 +70,10 @@ def id(id):
     return label(LabelType.ID, id)
 
 
+def manual(is_manual=True):
```

Comment:
why do we need argument `is_manual`? probably it can be like a simple tag (without calling)
`@allure.manual`
or
```
from allure import manual

@manual
def test():
   ...
```